virtual-organizations/EIC.yaml

```
@@ -0,0 +1,41 @@
+AppDescription: EIC collaboration.
+CertificateOnly: false
+Community: The EIC collaboration. 
+Contacts:
+  Administrative Contact:
+  - ID: acbb79b1cef1d22d7ca443ccd1aa9556750a8f3f
+    Name: Christopher Pinkenburg
+  Registration Authority:
+  - ID: acbb79b1cef1d22d7ca443ccd1aa9556750a8f3f
+    Name: Christopher Pinkenburg
+  - ID: 8a7d75ea65a2962520279197c72df6a16e1533a1
+    Name: John S. De Stefano Jr.
+  Security Contact:
+  - ID: acbb79b1cef1d22d7ca443ccd1aa9556750a8f3f
+    Name: Christopher Pinkenburg
+  VO Manager:
+  - ID: acbb79b1cef1d22d7ca443ccd1aa9556750a8f3f
+    Name: Christopher Pinkenburg
+Disable: false
+FieldsOfScience:
+  PrimaryFields:
+  - Nuclear Physics
+  SecondaryFields:
+  - Physics and astronomy
+ID: 137
```

```suggestion
ID: 139
```

Here's the ID that you want. Once you accept the suggestion we can get this merged.